src/js/modules/accounts/actions.js

```diff
@@ -0,0 +1,142 @@
+import axios from "axios";
+import appHistory from "../../tools/appHistory";
+
+import * as t from "./actionTypes";
+import { STUY_SPEC_API_URL, STUY_SPEC_API_HEADERS } from "../../constants";
+
+export const signUp = registrationParams => {
+  return dispatch => {
+    dispatch({
+      type: t.SIGN_UP_PENDING,
+      payload: registrationParams,
+    });
+
+    const deviseParams = (({ email, password, passwordConfirmation }) => ({
```

Comment:
Wait what. Why not object decomposition?
` const { email, password, passwordConfirmation } = registrationParams`

Reply:
It assists in describing what those things are for. Like, without that intermediate object, I wouldn't know that those are the params Devise accepts.